wolvic/wolvic_browser_context.cc

```diff
@@ -98,6 +98,11 @@ void WolvicBrowserContext::FinishInitWhileIOAllowed() {
   SimpleKeyMap::GetInstance()->Associate(this, key_.get());
   CreateUserPrefService();
 
+  if (off_the_record_) {
+    // Do not need to initialize visitedlink_writer in off the record context.
+    return;
+  }
+
```

Comment:
Hmm I prefer to move the writer initialization to a `if (!off_the_record)`. It's safer because people might add code after that and the early return will prevent it from running